### PR TITLE
Linux 2022 fixes

### PR DESCRIPTION
**Pull request by StephanWidor**
- readd `#endif`that somehow got lost in merge of master branch
- add stddef.h include to WDL so that gcc knows `NULL` and `size_t`
-  fix resources path for standalone app

### CODE REVIEW

**Comment by olilarkin** [comment]
Thanks for this. I don't like to modify the WDL files if possible. I wonder if that include can be fixed elsewhere, or submitted as a fix to [WDL](https://github.com/justinfrankel/WDL) which we then roll into iPlug2

**Comment by StephanWidor** [comment]
> Thanks for this. I don't like to modify the WDL files if possible. I wonder if that include can be fixed elsewhere, or submitted as a fix to [WDL](https://github.com/justinfrankel/WDL) which we then roll into iPlug2

Good point, wasn't aware of that! I made a PR for WDL. How would that go in here then? Just copy&paste?
As a less intrusive alternative, stddef could also be included in IGraphicsLinux.cpp, just before including wdlutf8.h. What do you think?

**Comment by olilarkin** [comment]
I think it should be fixed in IGraphicsLinux.cpp, IIRC i had a conversation with justin a long time ago about a similar thing

**Comment by StephanWidor** [comment]
> I think it should be fixed in IGraphicsLinux.cpp, IIRC i had a conversation with justin a long time ago about a similar thing

ok, removed the include from wdltypes.h, and added it to IGraphicsLinux.cpp.

File: Scripts/cmake/APP.cmake [hunk 1]
```diff
@@ -120,9 +120,8 @@ iplug_source_tree(iPlug2_APP)
 macro(iplug_configure_app target)
   iplug_target_add(${target} PUBLIC LINK iPlug2_APP)
 
-  set(res_dir "${CMAKE_BINARY_DIR}/${PLUG_NAME}.app/Contents/Resources")
-
   if (WIN32)
+    set(res_dir "${CMAKE_BINARY_DIR}/${PLUG_NAME}.app/Contents/Resources")
```
Comment (olilarkin) [inline Scripts/cmake/APP.cmake:124]: This looks weird for WIN32

Comment (StephanWidor) [reply Scripts/cmake/APP.cmake:124]: yep, was also a bit astonished, but this is what was set for win also before. I didn't want to change anything here, as I don't have a windows machine to test.

Comment (olilarkin) [reply Scripts/cmake/APP.cmake:124]: i removed it for now

**Pull request merged**